src/_layouts/default.html

```diff
@@ -260,7 +260,7 @@ <h2 class="title is-2">{{site.data.content["section-partners"].title[page.lang]}
             </figure>
           </div>
           <div>
-            <p>{{partner.name}}</p>
+            <!-- <p>{{partner.name}}</p> -->
```

Comment:
I think your version of master was outdated. 
```

<<<<<<< content-edits -- Your change
          <div>
            <!-- <p>{{partner.name}}</p> -->
          </div>
======= master
          {%- comment -%}
          <!-- <div>
            <p>{{partner.name}}</p>
          </div> -->
          {%- endcomment -%}
>>>>>>> 
```

The `div>p>{{partner.name}}` was not present in master. So you created an conflict. The version in master is better, since it does not leave the empty `div` in the source. Please resolve and use the version from master.

Reply:
ough, alright. Thanks for checking. Will do it later today